python/ray/workflow/tests/utils.py

```diff
@@ -25,11 +21,7 @@ def check_global_mark(name="workflow"):
 
 
 def _alter_storage(new_storage):
```

Comment:
should we just delete this method?

Reply:
yes, but I will leave it for the future to avoid linter errors. otherwise I have to cleanup those skipped tests